src/Microsoft.AspNet.OData.Shared/Builder/EdmTypeBuilder.cs

```diff
@@ -26,6 +26,7 @@ internal class EdmTypeBuilder
         private readonly Dictionary<PropertyInfo, IEdmProperty> _properties = new Dictionary<PropertyInfo, IEdmProperty>();
         private readonly Dictionary<IEdmProperty, QueryableRestrictions> _propertiesRestrictions = new Dictionary<IEdmProperty, QueryableRestrictions>();
         private readonly Dictionary<IEdmProperty, ModelBoundQuerySettings> _propertiesQuerySettings = new Dictionary<IEdmProperty, ModelBoundQuerySettings>();
+        private readonly Dictionary<IEdmProperty, PropertyConfiguration> _propertyConfigurations = new Dictionary<IEdmProperty, PropertyConfiguration>();
```

Comment:
>_propertyConfigurations [](start = 73, length = 23)

We can't use the "_properties" field?

Reply:
We need PropertyConfiguration. _properties does not have any configuration. 

---
In reply to: [358418941](https://github.com/OData/WebApi/pull/1977#discussion_r358418941) [](ancestors = 358418941)